cache-config/testing/docker/ort_test/Dockerfile

```diff
@@ -28,15 +28,17 @@
 # The recommended minimum size for each block devices is 1G.
 # For example, `sudo modprobe brd rd_size=1048576 rd_nr=2`
 
-FROM centos:7
+FROM centos:${RHEL_VERSION:-8}
 MAINTAINER dev@trafficcontrol.apache.org
 
+RUN yum -y install epel-release
+
+RUN yum repolist
+
 RUN yum install -y \
```

Comment:
Since these 3 `RUN` instructions all only perform `yum` commands, can they be condensed into a single `RUN` instruction for cache busting?

Reply:
done